client/driver/logging/rotator.go

```diff
@@ -26,7 +26,7 @@ const (
 	// lines when approaching the end of the file to avoid a log line being
 	// split between two files. Any single line that is greater than this limit
 	// may be split.
-	lineScanLimit = 16 * 1024
+	lineScanLimit = 32 * 1024
```

Comment:
❓ should the `logBufferSize` be increased to 64k so the scan won't read up the full buffer?

Reply:
Yes please

Reply:
@dadgar fixed